### DISCUSSION TITLE
Unable to uploading pdf larger than 1MB

### DISCUSSION THREAD

**Discussion by SharpLu** [danny-avila/LibreChat#3960] [Q&A]
Hi Community !

Hope you are doing well!
I have deployed the librechat helm (https://github.com/bat-bs/helm-charts/tree/main/charts/librechat) to kubernates, now everything is good!

I'm used the OpenAI model and when i upload the file larger than 1 MB, i got the error 
![Screenshot 2024-09-09 at 15 46 30](https://github.com/user-attachments/assets/7fe949b4-dc10-4036-bc79-c5b91d13b678)

But, when test file less than 1MB , everything is good!
![Screenshot 2024-09-09 at 15 47 21](https://github.com/user-attachments/assets/620fad3f-8105-4219-be67-13b65dcb6f08)

here is my config! 

Yaml config 
  # # For adding a custom config yaml-file you can set the contents in this var
  configYamlContent: |
    includedTools: []
    version: 1.1.5
    endpoints:
      custom:
        - name: 'TestAI'
          # For `apiKey` and `baseURL`, you can use environment variables that you define.
          # recommended environment variables:
          # Known issue: you should not use `OPENROUTER_API_KEY` as it will then override the `openAI` endpoint to use OpenRouter as well.
          apiKey: 'xx-xx!'
          baseURL: 'xxx:9099'
          models:
            default: ['custom_rag_1', 'openai_pipeline']
            fetch: true
          titleConvo: true
          titleModel: 'Custom RAG'
          # Recommended: Drop the stop parameter from the request as Openrouter models use a variety of stop tokens.
          dropParams: ['stop']
          modelDisplayLabel: 'Kindred AI'
    fileConfig:
      serverFileSizeLimit: 20  # Global server file size limit in MB
      avatarSizeLimit: 2  # Limit for user avatar image size in MB

also, tried https://www.librechat.ai/docs/configuration/librechat_yaml/object_structure/file_config
not really solve my problem !
Thank you

**Comment by danny-avila** [comment]
How are you resolving HTTPS? Likely an NGINX config issue: https://github.com/danny-avila/LibreChat/discussions/3061#discussioncomment-9776776

**Reply by SharpLu** [reply]
Hi @danny-avila  Thank you ! yes seems the client_max_body_size are the root cause for my case!. this discussion can be closed.
